org.gnome.EasyTAG.yaml

```diff
@@ -0,0 +1,122 @@
+id: org.gnome.EasyTAG
+runtime: org.gnome.Platform
+runtime-version: 3.28
+sdk: org.gnome.Sdk
+command: easytag
+finish-args:
+  # Full FS access in order to read media files.
+  - --filesystem=host
```

Comment:
At best I would stick to `xdg-music`, `xdg-download`, `/run/media` and `/media`. Users can modify these permissions after installation.

Reply:
Fixed by commit cfd4476.